draft-ietf-quic-manageability.md

```diff
@@ -605,9 +605,9 @@ A related question is whether the first packet of a given flow on a port known
 to be associated with QUIC is a valid QUIC packet.  This determination supports
 in-network filtering of garbage UDP packets (reflection attacks, random
 backscatter, etc.). While heuristics based on the first byte of the packet
-(packet type) could be used to separate valid from invalid first packet types,
-the deployment of such heuristics is not recommended, as bits in the first byte
-may have different meanings in future versions of the protocol.
+(packet type) could be used to distinguish garbage from non-garbage first packet
```

Comment:
why not just use non-valid?

Reply:
Actually this term just happened to be discussed between the IESG and RPC based on another case and in this context use of invalid seems fine. So maybe we just want to keep it.